core/modules/installer/installer.pages.inc

```diff
@@ -171,19 +171,19 @@ function installer_browser_installation_page($op) {
 
   switch ($op) {
     case 'select_versions':
-      backdrop_set_title(t("Select versions"));
+      backdrop_set_title(t('Select versions'));
       $content = installer_browser_installation_select_versions_page();
       break;
     case 'install_dependencies':
-      backdrop_set_title(t("Install Dependencies"));
+      backdrop_set_title(t('Install dependencies'));
       $content = installer_browser_installation_install_dependencies_page();
       break;
     case 'theme':
-      backdrop_set_title(t("Set themes"));
+      backdrop_set_title(t('Set themes'));
       $content = installer_browser_installation_enable_page('theme');
       break;
     case 'enable':
-      backdrop_set_title(t("Enable modules"));
+      backdrop_set_title(t('Enable modules'));
```

Comment:
...and before anyone asks, https://api.backdropcms.org/php-standards#quotes

> ... single quote strings are known to be faster because the parser doesn't have to look for in-line variables. Their use is recommended except in two cases: ...